pillar/vault/roles/macros.jinja

```diff
@@ -1,23 +1,21 @@
 {% macro pg_app_user(approle) %}
-        {% raw %}
         sql: >-
-          CREATE USER "{{name}}" WITH PASSWORD '{{password}}' VALID UNTIL '{{expiration}}';
-          GRANT "{{name}}" TO "bootcamp-ecommerce" WITH ADMIN OPTION;
-          GRANT "bootcamp-ecommerce" TO "{{name}}";
-          GRANT ALL PRIVILEGES ON ALL TABLES IN SCHEMA public TO "{{name}}";
-          GRANT ALL PRIVILEGES ON ALL SEQUENCES IN SCHEMA public TO "{{name}}";
-          ALTER DEFAULT PRIVILEGES IN SCHEMA public GRANT ALL PRIVILEGES ON TABLES TO "{{name}}";
-          ALTER DEFAULT PRIVILEGES IN SCHEMA public GRANT ALL PRIVILEGES ON SEQUENCES TO "{{name}}";
+          {% raw %}CREATE USER "{{name}}" WITH PASSWORD '{{password}}' VALID UNTIL '{{expiration}}';{% endraw %}
```

Comment:
Why the change from having the entire block using `raw` to one per line?

Reply:
So that we can use the approle parameter in the rendered Jinja